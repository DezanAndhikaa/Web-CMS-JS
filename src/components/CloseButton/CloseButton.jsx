import React from 'react';
import './CloseButton.scss';

export default class CloseButton extends React.PureComponent {
  render() {
    return (
      <div className="close-button-row">
        <div className={this.props.idBtnClose === "NonConfirmModal" ? "btn-close" : "close-button"} onClick={this.props.onClose} />
      </div>
    );
  }
}