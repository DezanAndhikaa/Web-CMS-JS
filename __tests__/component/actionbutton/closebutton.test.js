import React from 'react';
import Enzyme, { shallow } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import CloseButton from "../../../src/components/ActionButton/CloseButton/CloseButton";

Enzyme.configure({adapter:new Adapter()});
it("render close button correctly", () => {
  const wrapper = shallow (
    <CloseButton />
  )
  expect(wrapper).toMatchSnapshot();
})