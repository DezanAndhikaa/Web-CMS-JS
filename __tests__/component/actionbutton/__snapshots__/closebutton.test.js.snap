exports[`render close button correctly 1`] = `
<div
  className="close-button-row"
>
  <div
    className="close-button"
  />
</div>
`;
